libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -3291,6 +3291,20 @@ not_ ::
   Term fn Bool
 not_ = app notFn
 
+and_ ::
```

Comment:
rename `and_` and `or_` `(&&.)` and `(||.)` with the proper precedence to maintain coherence with the rest of the library.